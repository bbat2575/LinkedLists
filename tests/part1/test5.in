NEW 5

 
  
   
TYPE 0
NEW 5
 1
1 
0 0
 0
0 
TYPE 1
NEW 9
 2.2
2.2 
2 .2
2. 2
-2 .22
+2 .22
- 2.22
+ 2.22
2.2 2
TYPE 2
NEW 5
 a
  a
a 
a  
hell o  o   o    o     o     o    o   o  t here
TYPE 3