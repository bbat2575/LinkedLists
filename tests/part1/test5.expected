
Type HELP for a list of options
Press <CTRL + D> to Quit

> List 0:  ->   ->    ->     -> TYPE 0

> List 1: 1 -> 1 -> 0 -> 0 -> 0

> int -> int -> int -> int -> int

> List 2: 2.20 -> 2.20 -> 2.00 -> 2.00 -> -2.00 -> 2.00 -> 0.00 -> 0.00 -> 2.20

> float -> float -> float -> float -> float -> float -> float -> float -> float

> List 3:  a ->   a -> a  -> a   -> hell o  o   o    o     o     o    o   o  t here

> string -> string -> string -> string -> string

> 
